src/components/HeaderDropdown.tsx

```diff
@@ -21,17 +24,24 @@ export default function HeaderDropdown() {
   ]
 
   return (
-    <DropdownMenu>
+    <DropdownMenu open={isHovered}>
       <DropdownMenuTrigger asChild>
         <Button
           size="sm"
           variant="default"
           className="bg-blue-600 hover:bg-blue-700"
+          onMouseEnter={() => setIsHovered(true)} // Show dropdown on hover
+          onMouseLeave={() => setIsHovered(false)} // Hide dropdown when not hovering
+          onClick={() => navigate("/editor")} // Navigate on click
```

Comment:
it should take you to the quickstart, not the editor

```suggestion
          onClick={() => navigate("/quickstart")} // Navigate on click
```

Reply:
fixed